app/src/main/java/grinnell/appdev/edu/lyles/MenuViewPagerAdapter.java

```diff
@@ -0,0 +1,57 @@
+package grinnell.appdev.edu.lyles;
+
+import android.content.Context;
+import android.support.v4.app.FragmentManager;
+import android.support.v4.app.Fragment;
+import android.support.v4.app.FragmentStatePagerAdapter;
+
+import grinnell.appdev.edu.lyles.fragments.DrinksFragment;
+import grinnell.appdev.edu.lyles.fragments.FavoritesFragment;
+import grinnell.appdev.edu.lyles.fragments.HotFoodFragment;
+import grinnell.appdev.edu.lyles.fragments.SnacksFragment;
+
+/**
+ * Created by Mattori on 9/25/16.
```

Comment:
It's nice to know that the class was created by Mattori but you wanna tell us a bit about what it does? 
eg. `A {@code FragmentStatePagerAdapter} to handle the display of the various {@code Fragment}s ` On the next line, you can add the info about creation